gdeploy/gdeploy

```diff
@@ -83,7 +83,8 @@ def check_ansible_installation():
     dist_type = dist()
     op_system = dist_type[0]
     if op_system != 'redhat':
-        return "Error: gdeploy is supported on RHEL6 or above"
+        print "A non-Redhat Operating System is detected. No further checks will be executed"
```

Comment:
@chawlanikhil24 Nikhil, one minor nit pick, it is written as `Red Hat' or `redhat' not Redhat :-)
Please make that change, I'll merge it.